pkg/assets/create/creater.go

```diff
@@ -53,8 +48,8 @@ func EnsureManifestsCreated(ctx context.Context, manifestDir string, restConfig
 		return err
 	}
 
-	if options.Verbose && options.StdErr == nil {
-		options.StdErr = os.Stderr
+	if options.Logger == nil {
```

Comment:
If the options.Logger is not set, just don't log anything or create some dummy implementation of Logger interface?

Reply:
> If the options.Logger is not set, just don't log anything or create some dummy implementation of Logger interface?

The default I'm plugging in in this block [is a no-op logger][1] unless the user set that package-level global.

[1]: https://github.com/go-log/log/blob/a514cf01a3eb56ec6aff9ae5663453d990e68ed3/log.go#L17